LimeBean.Tests/BeanTests.cs

```diff
@@ -114,6 +114,17 @@ public class BeanTests {
         }
 
         [Fact]
+        public void Columns() {
+            var bean = new Bean();
+            bean["a"] = 1;
+            bean["b"] = 2;
+            bean["c"] = null;
+
+            var expect = new[] { "a", "b", "c" };
```

Comment:
The order of keys in dictionary is non-deterministic. Instead of comparing sequences, use LINQ's Intersect or Except, or anything similar.